### DISCUSSION TITLE
slope_calibration

### DISCUSSION THREAD

**Discussion by SyJin-pro** [IBM/aihwkit#646] [Q&A]
What does slope_calibration mean? How should I use this parameter?

**Comment by maljoras-sony** [comment]
Hi @123456789a12, 
Many thanks for the question. This is a somewhat experimental parameter that counter-acts approximately the drop-off in the I-O slope change when enabling using r-series or other forward-pass non-linearities. If for a sigmoidal non-lienarity is applied, you might want to change the slope of the sigmoid to correct on average for it. If you are interested in more details, please refer to the implementation, where the [non-linearity factor is computed](https://github.com/IBM/aihwkit/blob/69e524fbc2c5462cbe1a12ee1662879c001c30c0/src/rpucuda/rpu_forward_backward_pass.cpp#L298) which is then applied to the output, see [here](https://github.com/IBM/aihwkit/blob/69e524fbc2c5462cbe1a12ee1662879c001c30c0/src/rpucuda/rpu_forward_backward_pass.cpp#L693)
